demo/Views/DateTimePickerView.vala

```diff
@@ -41,8 +44,29 @@ public class DateTimePickerView : Gtk.Grid {
 
         var time_label = new Gtk.Label ("TimePicker:");
         time_label.halign = Gtk.Align.END;
-
+        
         timepicker = new Granite.Widgets.TimePicker ();
+        
+        show_seconds = new Gtk.Switch ();
+        show_seconds.halign = Gtk.Align.START;
+        show_seconds.set_tooltip_text ("Show Seconds");
+        show_seconds.notify["active"].connect (() => {
```

Comment:
This is probably better as a [Property Binding](https://valadoc.org/gobject-2.0/GLib.Object.bind_property.html)